src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #fff;
+}
+
+.header {
+  min-width: 1024px;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.icon {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  height: 100%;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 60px;
```

Comment:
it's better not set such a big line-height in case we don't know how many words or word lines will be. So use 
```
display: inline-block;
padding: 23px 0;
```